internal/core/input.rs

```diff
@@ -104,6 +104,8 @@ pub enum InputEventFilterResult {
     /// The Event will not be forwarded to children, if a children already had the grab, the
     /// grab will be cancelled with a [`MouseEvent::MouseExit`] event
     Intercept,
+    // TODO: add docs
```

Comment:
yeah, docs would be nice :-)  in particular, what's exactly the difference with ForwardAndInterceptGrab

Reply:
Yep. That's a good question. I originally wanted to just use `ForwardAndInterceptGrab` but this just caused `input_event` of `Flickable` not being called. The difference here is that the intercept is also happening in case of an aborted visit children result. See https://github.com/levrik/slint/blob/issue1107/internal/core/input.rs#L490-L492.
TBH I couldn't really figure out why `has_aborted()` was `true` in that case. I'm also unsure what `ForwardAndInterceptGrab` exactly does. There might be a solution which doesn't require a new input event filter result.